behat/tests/bootstrap/Common/AuthenticationTrait.php

```diff
@@ -41,6 +41,26 @@ public function iAmAtAdminLogin()
         $this->visitAdminPath('/login');
     }
 
+    /**
+     * @When /^I open the (admin |)(activation|password reset) page for "(.+)"$/
+     */
+    public function openActivationOrPasswordResetPage($admin, $pageType, $email)
+    {
+        $url = sprintf('/admin/fixtures/user-registration-token?email=%s', $email);
+        $this->iAmLoggedInToAdminAsWithPassword('admin@publicguardian.gov.uk', 'Abcd1234');
```

Comment:
Don't know why I didn't think about putting the login step in the helper functions that need it 🤦‍♂